crypto/nrf_cc310_platform/CHANGELOG.rst

```diff
@@ -9,6 +9,67 @@ Changelog - nrf_cc3xx_platform
 
 All notable changes to this project are documented in this file.
 
+nrf_cc3xx_platform - 0.9.16
+***************************
+
+New version of the library with the following features:
+
+* Added code-size optimized API for SHA-256 calculation in RAM and flash.
+* Fixed a bug where platform mutexes may end up unallocated if a context holding it is not zeroized.
+
+Library built against Mbed TLS version 3.1.0.
+
+Added
+=====
+
+Added a new build of nrf_cc3xx_mbedcrypto libraries for nRF9160, nRF52840, and nRF5340.
```

Comment:
It's stated here that a library for 52 is added.

But below we just add for 5340 and 9160.

Reply:
Will clean this up